spec/ic.did

```diff
@@ -35,6 +35,68 @@ type http_request_error = variant {
   conn_timeout;
 };
 
+type satoshi = nat64;
+
+type bitcoin_network = variant {
+  mainnet;
+  testnet;
+};
+
+type bitcoin_address = text;
+
+type block_hash = blob;
+
+type outpoint = record {
+  txid : blob;
+  vout : nat32
+};
+
+type utxo = record {
+  outpoint: outpoint;
+  value: satoshi;
+  height: nat32;
+};
+
+type pagination = record {
+  tip_block_hash : block_hash;
+  offset: opt nat32;
+  limit: opt nat32;
+};
+
+type get_utxos_request = record {
+  address : bitcoin_address;
+  network: bitcoin_network;
+  filter: opt variant {
+    min_confirmations: nat32;
+    pagination: pagination;
+  };
+};
+
+type get_utxos_error = variant {
+  MalformedAddress;
```

Comment:
@THLO I think we need to add this new error type and document it in the spec: https://github.com/dfinity/ic/blob/master/rs/bitcoin/types/public/src/lib.rs#L53